server/resources.py

```diff
@@ -44,7 +44,7 @@ def post(self):
 
         new_user = UserModel(
             email = data['email'],
-            credentials = data['credentials']
+            credentials = data['credentials'],
```

Comment:
looks like you need to rebase your code off the latest origin/dev - i fixed this one yesterday?

Reply:
Done - thanks!